src/js/entities-service/entities/actions.js

```diff
@@ -66,45 +74,36 @@ const _Model = BaseModel.extend({
       },
     });
   },
-  saveClinician(clinician) {
-    return this.save({ _clinician: clinician.id, _role: null }, {
-      relationships: {
-        clinician: this.toRelation(clinician),
-      },
-    });
-  },
-  saveRole(role) {
-    return this.save({ _role: role.id, _clinician: null }, {
+  saveOwner(owner) {
+    return this.save({
+      _owner: owner,
```

Comment:
This is making me wonder if we should bite the bullet and make all single relationships `{ id, type }` in the near future